analyzers/its/ReviewDiffs/packages.lock.json

```diff
@@ -1,13 +0,0 @@
-{
```

Comment:
@costin-zaharia-sonarsource This file gets regenerated when building the project and submitted to `git`. Shall it be restored?

Reply:
No, I think we can ignore also this one. It's not part of the packages we ship.

Reply:
Will you update .gitignore accordingly? Probably from
```
/analyzers/its/sources/**/packages.lock.json
```
to
```
/analyzers/its/**/packages.lock.json
```

Reply:
Yes, will do so.